cmd/git/git.go

```diff
@@ -578,6 +592,12 @@ func (j *DSGit) ParseArgs(ctx *shared.Ctx) (err error) {
 		j.SkipCacheCleanup = skipCacheCleanup
 	}
 
+	// setup stream
+	j.Stream = ctx.Env("GIT_STREAM")
```

Comment:
Can you change to `DA_GIT_STREAM` to be consistent? We use `DA_GIT_` prefix for all `git` env variables.

Reply:
done. updated.